<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VisWin</class>
 <widget class="QMainWindow" name="VisWin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1534</width>
    <height>870</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>200</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <widget class="iAFast3DMagicLensWidget" name="qvtkWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>100</width>
         <height>100</height>
        </size>
       </property>
      </widget>
      <widget class="QWidget" name="widget" native="true">
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QSlider" name="sStage">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget_2" native="true">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="leftMargin">
            <number>0</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Interval (msec):</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="sbInterval">
             <property name="minimum">
              <number>200</number>
             </property>
             <property name="maximum">
              <number>5000</number>
             </property>
             <property name="value">
              <number>1500</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbFirst">
             <property name="icon">
              <iconset resource="../../iAnalyse.qrc">
               <normaloff>:/images/4DCT_First.png</normaloff>:/images/4DCT_First.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbPrevious">
             <property name="icon">
              <iconset resource="../../iAnalyse.qrc">
               <normaloff>:/images/4DCT_Previous.png</normaloff>:/images/4DCT_Previous.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbPlay">
             <property name="icon">
              <iconset resource="../../iAnalyse.qrc">
               <normaloff>:/images/4DCT_Play.png</normaloff>:/images/4DCT_Play.png</iconset>
             </property>
             <property name="checkable">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbNext">
             <property name="icon">
              <iconset resource="../../iAnalyse.qrc">
               <normaloff>:/images/4DCT_Next.png</normaloff>:/images/4DCT_Next.png</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="pbLast">
             <property name="icon">
              <iconset resource="../../iAnalyse.qrc">
               <normaloff>:/images/4DCT_Last.png</normaloff>:/images/4DCT_Last.png</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>32</width>
     <height>32</height>
    </size>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextBesideIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionResetCam"/>
   <addaction name="actionXYView"/>
   <addaction name="actionXYBackView"/>
   <addaction name="actionXZView"/>
   <addaction name="actionXZBackView"/>
   <addaction name="actionYZView"/>
   <addaction name="actionYZBackView"/>
   <addaction name="actionAddToMagicLens"/>
   <addaction name="actionMagicLens"/>
  </widget>
  <action name="actionAddToMagicLens">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Add to Magic Lens</string>
   </property>
   <property name="toolTip">
    <string>ADD TO MAGIC LENS</string>
   </property>
  </action>
  <action name="actionMagicLens">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Magic Lens</string>
   </property>
   <property name="toolTip">
    <string>MAGIC LENS</string>
   </property>
  </action>
  <action name="actionResetCam">
   <property name="text">
    <string>Reset Camera</string>
   </property>
  </action>
  <action name="actionXYView">
   <property name="text">
    <string>XY</string>
   </property>
  </action>
  <action name="actionYZView">
   <property name="text">
    <string>YZ</string>
   </property>
  </action>
  <action name="actionXZView">
   <property name="text">
    <string>XZ</string>
   </property>
  </action>
  <action name="actionXYBackView">
   <property name="text">
    <string>-XY</string>
   </property>
  </action>
  <action name="actionXZBackView">
   <property name="text">
    <string>-XZ</string>
   </property>
  </action>
  <action name="actionYZBackView">
   <property name="text">
    <string>-YZ</string>
   </property>
   <property name="toolTip">
    <string>-YZ</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>iAFast3DMagicLensWidget</class>
   <extends>QWidget</extends>
   <header>iAFast3DMagicLensWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../iAnalyse.qrc"/>
 </resources>
 <connections/>
</ui>
